docsrc/user_guide/dynamo_export.rst

```diff
@@ -0,0 +1,72 @@
+.. _dynamo_export:
+
+Torch-TensorRT (Dynamo) Backend
+========================
+This guide presents Torch-TensorRT dynamo backend which compiles Pytorch programs 
+into TensorRT engines through torch dynamo. Pytorch 2.1 introduced ``torch.export`` APIs which 
+can export graphs from Pytorch programs using torch dynamo. Torch-TensorRT dynamo 
+backend compiles these exported graphs and optimizes them using TensorRT. Here's a simple 
+usage of the dynamo backend  
+
+.. code-block:: python
+
+    import torch
+    import torch_tensorrt
+
+    model = MyModel().eval().cuda()
+    inputs = torch.randn((1, 3, 224, 224), dtype=torch.float32).cuda()
+    exp_program = torch.export(model, inputs)
+    trt_gm = torch_tensorrt.dynamo.compile(exp_program, inputs) # Output is a torch.fx.GraphModule
+    trt_gm(inputs)
```

Comment:
These could need modification if the above `inputs` is changed to a tuple

Reply:
done